src/VolumeLogger.js

```diff
@@ -59,6 +59,295 @@ module.exports = class VolumeLogger {
+                    }
+
+                    if (sec === 1) {
+                        minReport.peerName = secReport.peerName
+                        minReport.startTime = secReport.startTime
+                        minReport.currentTime = secReport.currentTime
+                        minReport.timestamp = 0
+
+                        minReport.broker.messagesToNetworkPerSec = secReport.broker.messagesToNetworkPerSec
+                        minReport.broker.bytesToNetworkPerSec = secReport.broker.bytesToNetworkPerSec
+
+                        minReport.network.avgLatencyMs = secReport.network.avgLatencyMs
+                        minReport.network.bytesToPeersPerSec = secReport.network.bytesToPeersPerSec
+                        minReport.network.bytesFromPeersPerSec = secReport.network.bytesFromPeersPerSec
+                        minReport.network.connections = secReport.network.connections
+
+                        if (metricsReport.metrics['broker/cassandra']) {
+                            minReport.storage.bytesWrittenPerSec = secReport.storage.bytesWrittenPerSec
+                            minReport.storage.bytesReadPerSec = secReport.storage.bytesReadPerSec
+                        }
+                    } else {
+                        minReport.broker.messagesToNetworkPerSec = throtheledAvg(minReport.broker.messagesToNetworkPerSec, secReport.broker.messagesToNetworkPerSec)
+                        minReport.broker.bytesToNetworkPerSec = throtheledAvg(minReport.broker.bytesToNetworkPerSec, secReport.broker.bytesToNetworkPerSec)
+
+                        minReport.network.avgLatencyMs = throtheledAvg(minReport.network.avgLatencyMs, secReport.network.avgLatencyMs)
+                        minReport.network.bytesToPeersPerSec = throtheledAvg(minReport.network.bytesToPeersPerSec, secReport.network.bytesToPeersPerSec)
+                        minReport.network.bytesFromPeersPerSec = throtheledAvg(minReport.network.bytesFromPeersPerSec, secReport.network.bytesFromPeersPerSec)
+                        minReport.network.connections = throtheledAvg(minReport.network.connections, secReport.network.connections)
+
+                        if (metricsReport.metrics['broker/cassandra']) {
+                            minReport.storage.bytesWrittenPerSec = throtheledAvg(minReport.storage.bytesWrittenPerSec, secReport.storage.bytesWrittenPerSec)
+                            minReport.storage.bytesReadPerSec = throtheledAvg(minReport.storage.bytesReadPerSec, secReport.storage.bytesReadPerSec)
+                        }
+                    }
+
+                    securePublish(
+                        this.streamIds.secStreamId,
+                        secReport
+                    )
+
+                    const now = Date.now()
+
+                    if (sec === 60 || (minReport.timestamp + (60 * 1000) - now) < 0) {
+                        minReport.timestamp = secReport.currentTime
```

Comment:
Would it be better to `minReport = Date.now()` or is this being set intentionally to mirror `secReport.currentTime`?